src/app/(nousSuivre)/actions-solidaires/page.js

```
@@ -180,22 +188,23 @@ export default function Actions() {
                 animate="visible"
                 variants={fadeIn}
-                className="bg-yellow-200 p-8"
+                className=" p-8" style={{ backgroundColor: "#F8D24D" }}
             >
-                <div className="items-center gap-8 xl:px-80">
+                <div className="items-center gap-8 xl:px-20">
                     <div className="text-red-700 text-2xl md:text-4xl font-bold my-8">
                         Actions <br /> Diverses
                     </div>
-                    <p> Le pôle Actions Solidaires propose divers services sociaux, incluant la circoncision, le soutien aux familles pendant l'Aid al-Adha, et inculquer à nos entants du foyer Bab Rayan nos valeurs solidaires par le biais de partage de dons en nature, reçus de nos partenaires et donateurs, aux familles en besoins.
+                    <p> Le pôle Actions Solidaires propose divers services sociaux, incluant la circoncision, le soutien aux familles pendant l'Aid al-Adha.
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)